.npmignore

```diff
@@ -0,0 +1,2 @@
+.babelrc
+.flowconfig
```

Comment:
I think an `.npmignore` file isn't necessary. Since the the `package.json` declares a `files` list, that means anything that isn't in that list won't be included.

Reply:
Ah I wasn't aware of the `files` key, will remove now.